gcloud-java-datastore/src/main/java/com/google/gcloud/datastore/DatastoreImpl.java

```diff
@@ -186,15 +194,46 @@ public Entity get(Key key) {
   }
 
   @Override
+  public Entity get(Key key, ReadOption... options) {
+    return DatastoreHelper.get(this, key, options);
+  }
+
+  @Override
   public Iterator<Entity> get(Key... keys) {
     return get(null, keys);
   }
 
   @Override
+  public Iterator<Entity> get(Iterable<Key> keys, ReadOption... options) {
+    return get(toReadOptionsPb(options), Iterables.toArray(keys, Key.class));
+  }
+
+  private static com.google.datastore.v1beta3.ReadOptions toReadOptionsPb(ReadOption... options) {
+    com.google.datastore.v1beta3.ReadOptions readOptionsPb = null;
+    if (options != null) {
+      Map<Class<? extends ReadOption>, ReadOption> optionsMap = ReadOption.asImmutableMap(options);
+      EventualConsistency eventualConsistency =
+          (EventualConsistency) optionsMap.get(EventualConsistency.class);
+      if (eventualConsistency != null) {
```

Comment:
Nit: this could be:
```java
if (options != null
    && ReadOption.asImmutableMap(options).containsKey(EventualConsistency.class)) {
  readOptionsPb = com.google.datastore.v1beta3.ReadOptions.newBuilder()
      .setReadConsistency(ReadConsistency.EVENTUAL)
      .build()
}
```